examples/simple_key_value_store.rs

```diff
@@ -71,8 +74,9 @@ Usage:
 
 Options:
   -h --help           Show this screen.
-  --type=<node_type>  node_type = client | node | first.
-  --peer=<end_point>  end_point
+  --client            node runs as a client
+  --first             node runs as the first non-client node in the network
+  --node              node runs as the non-first non-client node in the network
```

Comment:
Please use proper sentences for the descriptions (capital initial letter, full stop at end).  We probably need to explain somewhere the difference between client and node modes.